### PR TITLE
Add "it works" page

### PR DESCRIPTION
**Pull request by dominik-zeglen**
I want to merge this change because it adds "it works" page.

Screenshot 2020-04-22 at 02 18 38

# Impact

* [ ] New migrations
* [ ] New/Updated API fields or mutations
* [ ] Deprecated API fields or mutations
* [ ] Removed API types, fields, or mutations

# Pull Request Checklist

* [ ] Privileged queries and mutations are guarded by proper permission checks
* [ ] Database queries are optimized and the number of queries is constant
* [ ] Database migration files are up to date
* [ ] The changes are tested
* [ ] GraphQL schema and type definitions are up to date
* [x] Changes are mentioned in the changelog

### CODE REVIEW

**Comment by dominik-zeglen** [comment]
I need someone to serve this view, @IKarbowiak maybe?

**Comment by IKarbowiak** [comment]
@dominik-zeglen You should create a pull request to `feature/splash-page` where the view was created.

**Comment by dominik-zeglen** [comment]
Ah okay

File: templates/home.html [hunk 1]
```diff
@@ -1,19 +1,2962 @@
...
+            fill="#13BEBB"
+          />
+          <path
+            fill-rule="evenodd"
+            clip-rule="evenodd"
+            d="M8.0181 2.23959C8.13087 2.12714 8.28363 2.064 8.44288 2.064H28.5227C28.7662 2.064 28.9856 2.21076 29.0786 2.43577C29.1717 2.66078 29.1199 2.91967 28.9475 3.09159L23.1576 8.86492C23.0448 8.97736 22.892 9.04051 22.7328 9.04051H2.65297C2.40949 9.04051 2.19002 8.89375 2.09701 8.66874C2.00399 8.44372 2.05578 8.18483 2.22819 8.01291L8.0181 2.23959ZM8.69157 3.26718L4.1083 7.83732H22.4841L27.0674 3.26718H8.69157Z"
+            fill="#28234A"
+          />
+        </svg>
+        <span> <a href="#">View documentation</a> for Saleor 2.10 </span>
```
Comment (maarcingebala) [inline templates/home.html:209]:
```suggestion
        <span><a href="https://docs.saleor.io/" target="_blank">View documentation</a> for Saleor 2.10 </span>
```

Comment (dominik-zeglen) [reply templates/home.html:209]: Right, forgot about it

**Comment by maarcingebala** [comment]
@dominik-zeglen We need to do a few more changes:
- if `storefront_url` and `dashboard_url` variables are empty, we should display the message about saleor-platform (see the designs)
- make the layout a bit more responsive - nobody is going to run this on mobile or tablet, but small desktops should be supported (now it doesn't fit on a laptop screen).

**Comment by dominik-zeglen** [comment]
> * if `storefront_url` and `dashboard_url` variables are empty, we should display the message about saleor-platform (see the designs)

`and` or `or`? I think it should be `or`.

File: templates/home/index.html [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+            </div>
+
+            {% if not storefront_url and not dashboard_url %}
+            <p>
+              It looks like you're only running the Saleor core. If you're
+              looking for an easy way to run all services locally, check out
+              <a
+                href="https://github.com/mirumee/saleor-platform"
+                target="_blank"
+                >saleor platform.</a
```
Comment (maarcingebala) [inline templates/home/index.html:206]: I think it deserves it:
```suggestion
                >Saleor Platform.</a
```

Comment (dominik-zeglen) [reply templates/home/index.html:206]: Copied straight out of figma 🤷‍♂️

**Pull request merged**